mmm_experiments/adjudicators/msg.py

```diff
@@ -0,0 +1,40 @@
+from pydantic import BaseModel
+
+DEFAULT_NAME = "agent_suggestions"
+
+
+class Suggestion(BaseModel):
+    ask_uid: str  # UID from the agent ask message
+    plan_name: str
+    plan_args: list = []
```

Comment:
pydandic qustion: this is a list literal, the default for the next one in a function to create the dictionary, should this be `list` or should the next one be `{}` ?

Reply:
This is a very good question, that I will dive into. It came from intially using dataclasses with default factories, and not porting everything completely.

Reply:
https://stackoverflow.com/questions/63793662/how-to-give-a-pydantic-list-field-a-default-value